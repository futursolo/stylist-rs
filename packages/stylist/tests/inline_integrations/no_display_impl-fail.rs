enum NoDisplay {
    ND,
}
fn main() {
    let expr = NoDisplay::ND;
    let _ = stylist::css! {
        background: ${expr};
    };
}
