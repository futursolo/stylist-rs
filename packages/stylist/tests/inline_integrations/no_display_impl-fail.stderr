error[E0277]: `NoDisplay` doesn't implement `std::fmt::Display`
 --> tests/inline_integrations/no_display_impl-fail.rs:7:23
  |
7 |         background: ${expr};
  |                       ^^^^ `NoDisplay` cannot be formatted with the default formatter
  |
  = help: the trait `std::fmt::Display` is not implemented for `NoDisplay`
  = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
  = note: required for the cast from `NoDisplay` to the object type `dyn std::fmt::Display`
